.github/scripts/release.sh

```diff
@@ -26,6 +27,25 @@ while [[ $# -gt 0 ]]; do
       COMMIT=yes
       shift # pop option
       ;;
+    --azure-cdn-uri)
+      AZURE_CDN_URI=$(realpath "$2")
+      shift # pop option
+      shift # pop option
+      ;;
```

Comment:
This is references below, but the variable seems to be unused? Was this left over from a round of cleanup? :thinking:

Reply:
Correct, removed

Reply:
What about the references below? It now says things like `As --azure-cdn-uri flag not defined`, but there is no such flag anymore. Probably best to do another pass over the comments. :pray:

Reply:
nice catch, not on my game today it seems 🙈